src/job/job.spec.js

```diff
@@ -166,4 +166,40 @@ describe("job", () => {
       ).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".runSequence() :", () => {
+    afterEach(async () => {
+      await Promise.all(
+        xo.objects.all[config.vmIdXoTest].snapshots.map(id =>
+          xo
+            .call("vm.delete", { id, delete_disks: true })
+            .catch(error => console.error(error))
+        )
+      );
+    });
+
+    it("runs a job", async () => {
+      jest.setTimeout(15e3);
```

Comment:
Is setting a larger timeout necessary ?

Reply:
`10e3` is too small : sometimes it works, and sometimes not. 
But I will try with values between 10 and 15...